inst/templates/ad_advs.R

```diff
@@ -70,9 +71,9 @@ cdc_bmi_for_age <- cdc_bmiage %>%
 # Default reference sources: WHO for children <2 yrs old (< 730.5 days),
 # and CDC for children >=2 yrs old (>= 730.5 days)
 # Combine WHO and CDC metadata
-data(who_lgth_ht_for_age_boys)
-data(who_lgth_ht_for_age_girls)
-data(cdc_htage)
+who_lgth_ht_for_age_boys <- admiralpeds::who_wt_for_age_boys
+who_lgth_ht_for_age_girls <- admiralpeds::who_wt_for_age_girls
```

Comment:
Didn't you reverse the calls at lines 75 with lines 116 ?
e.g. should be `who_lgth_ht_for_age_girls <-admiralpeds::who_lgth_ht_for_age_girls`